share/picongpu/tests/CollisionsBeamRelaxation/include/picongpu/param/density.param

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2013-2021 Axel Huebl, Heiko Burau, Rene Widera, Felix Schmitt,
+ *                     Richard Pausch, Pawel Ordyna
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *
+ * Configure existing or define new normalized density profiles here.
+ * During particle species creation in speciesInitialization.param,
+ * those profiles can be translated to spatial particle distributions.
+ */
+
+#pragma once
+
+#include "picongpu/particles/densityProfiles/profiles.def"
+/* preprocessor struct generator */
+#include <pmacc/preprocessor/struct.hpp>
+
+
+namespace picongpu
+{
+    namespace SI
+    {
+        /** Base density in particles per m^3 in the density profiles.
+         *
+         * This is often taken as reference maximum density in normalized profiles.
+         * Individual particle species can define a `densityRatio` flag relative
+         * to this value.
+         *
+         * unit: ELEMENTS/m^3
+         */
+        // reference frequency like in smilei simulations
+        constexpr float_64 REFERENCE_OMEGA_SI = 2.0 * PI * 3.0e8 * 1.0e6;
+        // reference density from smilei simulations
```

Comment:
Well I scaled all parameters with `REFERENCE_OMEGA_SI` since that's how `smilei` is introducing absolute scales into similations. I wanted to make it easy to compare `smilei` and `PIConGPU` inputs because the setup is taken over from `smilei`'s benchmarks.

Reply:
Okay - I understand that. @ComputationalRadiationPhysics/picongpu-maintainers @bussmann @ax3l what do you think, shall we reference another PIC code?

Reply:
Why not, this test case based on the documentation in Smilei, therefore it schould be referenced.